sycl/plugins/esimd_emulator/pi_esimd_emulator.hpp

```diff
@@ -74,6 +74,8 @@ struct _pi_device : _pi_object {
       : Platform{ArgPlt}, CmDevicePtr{ArgCmDev}, VersionStr{ArgVersionStr} {}
 
   pi_platform Platform;
+  // TODO: Check if serialization is required when ESIMD_EMULATOR
+  // plug-in calls CM runtime functions
   cm_support::CmDevice *CmDevicePtr = nullptr;
```

Comment:
can it be & instead of * ?
I mean, for C++ using raw pointers when references can be used (no pointer reassignment/arithmetics/etc required) is questionalbe

Reply:
This is how other plug-ins are creating and managing device info.

Reply:
How do make a reference to nullptr?

Reply:
>> How do you make a reference to nullptr?

My question is: is nullptr needed in this case?

Reply:
Is there specific problem you are seeing with this code? Or is it just stylistic consideration? I don't think this should hold this PR then.

Reply:
I'll address this pointer/nullptr/reference problem in a separate PR if needed.